app/blueprints/templates/routes.py

```diff
@@ -12,9 +19,32 @@
     template_folder="templates",
 )
 
+file_upload_path = os.path.join(os.path.dirname(__file__), "uplaoded_files")
```

Comment:
DO we still need to upload & create a file inside the container cant we read the maltipartfile it self

Reply:
Thanks @nuwan-samarasinghe  removed this, this was added earlier to save file, which is not used now